dialogs/Android/Android-dialog.md

```diff
@@ -3,330 +3,320 @@ title: "Dialogs"
 layout: detail
 section: components
 excerpt: "Dialogs are modal windows that require interaction."
-iconId: 
-path: /catalog/dialog/
+iconId: dialog
+path: /catalog/dialogs/
 -->
 
-
 # Dialogs
 
-Dialogs inform users about a task and can contain critical information, require decisions, or involve multiple tasks.
+[Dialogs](https://material.io/components/dialogs/) inform users about a task and can contain critical information, require decisions, or involve multiple tasks.
 
 There are four types of dialogs:
 
 1. [Alert](#alert-dialog)
-1. [Simple](#simple-dialog)
-1. [Confirmation](#confirmation-dialog)
-1. [Full-screen](#full-screen-dialog)
+2. [Simple](#simple-dialog)
+3. [Confirmation](#confirmation-dialog)
```

Comment:
FYI, markdown automatically renumbers lists, so you don't need to renumber anything.